filesystems/nfs/connectathon/runtest.sh

```diff
@@ -586,16 +568,16 @@ else
     echo " ========== Use default single test run ============"
 fi
 
-if [ "$CTHONSERVERS" ]; then
+if [ -n "$CTHONSERVERS" ]; then
     # We can specify a list of servers it test against
     # For example, we can provide a customized list of
     # servers CTHONSERVER="host1-nfs host2-nfs host3-nfs"
     servers="$CTHONSERVERS"
     echo " ========== Override server list =========="
     echo " ***** INFO: CTHONSERVERS=$CTHONSERVERS *****"
 else
-    # Get the server list
-    getServerList
+    echo "server list should defined via env CTHONSERVERS"
+    exit 1
```

Comment:
Hi @idorax ,and @rasibley,
When the server list is not defined in CTHONSERVERS, do we need just abort the task or return "exit 1" ?

Reply:
@ftjahjad good idea, we should probably abort here since it's a setup error. @idorax thanks for letting me know, I noticed the kpet-db MR after the fact.

Reply:
Hi @ftjahjad, I've updated the code block according to your comments. Please help to take a look again, thanks! 

- http://pastebin.test.redhat.com/854073

Reply:
A beaker job `4189119` is done to test 'abort' if CTHONSERVERS is not defined.

Reply:
Hi @idorax Looks good to me.  and @rasibley  already merged the PR
Thanks for porting  it.